cfgov/wagtailadmin_overrides/static/css/global.css

```diff
@@ -14,12 +15,27 @@ body:before {
 @media screen and (min-width: 800px) {
   body:before {
     box-sizing: border-box;
-    width: 200px;
-    padding: 0.5em;
     position: fixed;
+    overflow: hidden;
+    z-index: 1000;
     top: 0;
     left: 0;
-    z-index: 100;
+    width: 150px;
+    height: 35px;
+    margin-top: 0.5em;
+    padding: 0.5em;
+    line-height: 1.5;
+    transition-duration: .15s;
```

Comment:
Doesn't appear we are running this file through linting (or does `yarn lint` do anything to this file?). It would likely update this to `0.15s`. We probably should include it in linting.

Reply:
Updated that line in [424684d](https://github.com/cfpb/consumerfinance.gov/pull/7683/commits/424684d4b87430c496af3c0f428dc7b56a0e8e3f).